build.sbt

```diff
@@ -0,0 +1,59 @@
+import Dependencies._
+import sbtrelease.ReleaseStateTransformations._
+
+ThisBuild / version := (ThisBuild / version).value
+ThisBuild / organization     := "uk.gov.nationalarchives"
+ThisBuild / organizationName := "National Archives"
+
+ThisBuild / scmInfo := Some(
+  ScmInfo(
+    url("https://github.com/nationalarchives/tdr-backend-checks-utils"),
+    "git@github.com:nationalarchives/tdr-backend-checks-utils.git"
+  )
+)
+developers := List(
+  Developer(
+    id    = "tna-digital-archiving-jenkins",
```

Comment:
We really should change that id and remove the `-jenkins` bit at some point.

Reply:
I know but that's what the user is called. We could just pretend it was someone called Jenkins.